src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -454,6 +470,250 @@ components:
+          description: The ID of the Prison that that Prisoner is currently at.
+        prisonName:
+          type: string
+          description: The name of the Prison that that Prisoner is currently at.
+        hopingToGetWork:
+          $ref: '#/components/schemas/HopingToWork'
+        desireToWork:
+          type: boolean
+          description: Whether the Prisoner wishes to work or not.
+        reasonToNotGetWorkOther:
+          type: string
+          description: The reason that is given when the Prisoner does not want to work. This is mandatory when 'reasonToNotGetWork' is set to 'OTHER'.
+        abilityToWork:
+          uniqueItems: true
+          type: array
+          description: One or more factors affecting the Prisoner's ability to work.
+          items:
+            $ref: '#/components/schemas/AbilityToWorkFactor'
+        abilityToWorkOther:
+          type: string
+          description: A specific factor affecting the Prisoner's ability to work. This is mandatory when 'abilityToWork' is set to 'OTHER'.
+        reasonToNotGetWork:
+          uniqueItems: true
+          type: array
+          description: One or more reasons the Prisoner has given not to get work after leaving Prison.
+          items:
+            $ref: '#/components/schemas/ReasonNotToWork'
+        workExperience:
+          "$ref": "#/components/schemas/PreviousWorkResponse"
+        skillsAndInterests:
+          "$ref": "#/components/schemas/SkillsAndInterestsResponse"
+        qualificationsAndTraining:
+          "$ref": "#/components/schemas/EducationAndQualificationResponse"
+        inPrisonInterests:
+          "$ref": "#/components/schemas/PrisonWorkAndEducationResponse"
+        createdBy:
+          type: string
+          description: The DPS username of the person who created the Induction.
+          example: 'asmith_gen'
+        createdDateTime:
+          type: string
+          format: date-time
+          description: An ISO-8601 timestamp representing when the Induction was created.
+          example: '2023-06-19T09:39:44Z'
+        modifiedBy:
+          type: string
+          description: The DPS username of the person who last updated the Induction.
+          example: 'asmith_gen'
+        modifiedDateTime:
+          type: string
+          format: date-time
+          description: An ISO-8601 timestamp representing when the Goal was last updated. This will be the same as the created date if it has not yet been updated.
+          example: '2023-06-19T09:39:44Z'
+      required:
+        - reference
+        - offenderId
+        - hopingToGetWork
+        - createdBy
+        - createdDateTime
+        - modifiedBy
+        - modifiedDateTime
+    PreviousWorkResponse:
+      title: PreviousWorkResponse
+      description: A Prisoner's previous work experience. Migrated from hmpps-ciag-careers-induction-api.
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
```

Comment:
I've changed the format of the id from an integer to a UUID. I'm expecting it to break the CIAG UI (though perhaps not at runtime?), but this is something we can easily fix when we port the UI to call our API.

Reply:
👍 , agreed, as discussed yesterday 👍